dashboard/src/app/workspaces/share-workspace/user-item/user-item.html

```diff
@@ -19,16 +20,16 @@
            class="che-list-item-name">
         <span class="che-xs-header noselect" hide-gt-xs>Email</span>
         <span><img class="user-face" gravatar-src="userItemController.user.email"></span>
-        <span class="user-email che-hover ">{{userItemController.user.email}}</span>
+        <span class="user-email che-hover " name="member-name">{{userItemController.user.email}}</span>
```

Comment:
Could you rename  "member-name"  to  "member-email"

Reply:
Also I would notice that it is about _user_, not _member_.

Reply:
`name` attribute is usually used in **input** elements. It's better to use `id` for **div** element.